### DISCUSSION TITLE
Is it possible to quote a post in a repost  without generating a reply event to the original account who did the post?

### DISCUSSION THREAD

**Discussion by MatthiasThoemel** [MarshalX/atproto#411] [Questions] [answered]
Hi,

the BlueSky Community Guidlines forbid spamming. Especially if the automated sending of a post / repost are generating disturbing messages on other accounts, like a reply-message because of the quote:

Text from BlueSky Community Guidelines:
"Generating automated or bulk interactions, including any that would cause a notification to a user like a message, follow, like or reply"

https://bsky.social/about/support/community-guidelines

Is it possible with the AT protocoll to do a repost with quoting but without genering a reply message on the account who did the original post?

If not, I have to stop my BOT unfortunately. 

Or I must do a trick: Do a copy of the original post and use this as an image with a link to the original account. But I would like not to do it like that.

My best regards, Matthias

**Comment by MarshalX** [comment]
Hi,

Probably you can investigate around detached quotes. I am not sure does it produce notification or not. Otherwise, you always can just copy original post and link to the original source. But this is a huge ugly workaround

**Reply by MatthiasThoemel** [reply]
Thank you for your suggestions. 

I have not enough knowlege about the AT protocol and the object structure of BlueSky so I do not find any good hint how to detach a quote. I found some small object information about detach and frozen but that is not enough to test it.

Also your idea of copying the original post I cannot test because I do not know how to link the copy to the original post.

I tried to find a python webview renderer to do a 'picture' of the original to use it in my Wiki BOT post, but it seems to be that the pywebview  modules are a kind of 'old'. They do not work on my computer. 

I could open a chrome browser and use selenium to do a screen shot of the original post, maybe I will think about this way.

Thank you again!!!

**Reply by MatthiasThoemel** [reply]
Hi, I did it like I described it yesterday: I am rendering an image of the original post by using selenium and this picture I am using to post the Wikipedia links related to that post.

To get 100 percent I would like to put a link on the picture which will lead to the original post. I did already a link in the header but that is not intuitive.

I did not figured out how to do this with atproto. Could you please help me in that case again?

![image](https://github.com/user-attachments/assets/d0be8468-7ba6-47d6-b713-67d121f356ec)

**Reply by MarshalX** [reply]
Hi @MatthiasThoemel,

I am a bit of lost. What exactly do you mean by "put a link on the picture"? Do you mean alt text?

**Reply by MatthiasThoemel** [reply]
Hi, I mean: if the reader touches the picture, it should lead (by a public https link) to the original post. If you now touch the picture, Bluesky just shows the picture in a bigger version. Is it possible to change this behavioral the reader will see the original post if he or she touches the picture?

**Reply by MarshalX** [reply, marked as answer]
Probably you mean website embeds in posts with Open Graph data like thumbnail, title, description... And we have the exact example https://github.com/MarshalX/atproto/blob/main/examples/advanced_usage/send_ogp_link_card.py. You can try use link as post`s link and thumbnail as your screenshot

**Reply by MatthiasThoemel** [reply]
Thanks for this idea. I realized it and faced a little disadvantage: the thumbnail has a maximal heigh, so in my case I resize my pictures to an accepted aspect ratio. If the original picture is very tal, it results in that, which is OK in a way:

![image](https://github.com/user-attachments/assets/61c994bc-cd9f-41c5-a199-22cb49119511)

This picture fits good:

![image](https://github.com/user-attachments/assets/5dbc97d4-0b80-405c-845d-53d4f7483f33)

Thank you very much for your help.

**Reply by MarshalX** [reply]
@MatthiasThoemel could we mark this question as answered?

**Reply by MatthiasThoemel** [reply]
Good Morning, I marked your Answer - you can close this ticket - thank you.
